cmd/build/helmify/static/README.md

```diff
@@ -166,7 +166,8 @@ information._
 | emitAuditEvents                                            | Emit K8s events in configurable namespace for audit violations (alpha feature)                                                                                                                                                                                                                 | `false`                                                                                                                                                               |
 | enableK8sNativeValidation                                   | Enable the K8s Native Validating driver to create CEL-based rules  (alpha feature)                                                                                                                                                                                                                 | `false`                                                                                                                                                               |
-| vapEnforcement                                              | Generate K8s Validating Admission Policy resource. Allowed values are NONE: do not generate, GATEKEEPER_DEFAULT: do not generate unless label gatekeeper.sh/use-vap: yes is added to policy explicitly, VAP_DEFAULT: generate unless label gatekeeper.sh/use-vap: no is added to policy explicitly.  (alpha feature)                                                                                                                                                                                                                 | `GATEKEEPER_DEFAULT`                                                                                                                                                               |
+| vapEnforcement                                              | Generate K8s Validating Admission Policy resource. Allowed values are false: do not generate unless generateVAP: true is added to constraint template explicitly, true: generate unless generateVAP: false is added to constraint template explicitly. (beta feature)                                                                                                                                                                                                                 | `false`                                                                                                                                                               |
```

Comment:
Beta or alpha?

I thought we were going beta with the language? May be misremembering the mtg where this was discussed.

Reply:
I see Rita says beta, if it was settled in the mtg, fine, but I vaguely recall us saying the engine was beta, not the generation.

Reply:
I do remember we decided to move to `beta` but we probably need better wording around flag name and description to accratly indicate what it does.

Reply:
You are right. lets update the feature as beta in the doc and keep the generation of vap flags as alpha.

Reply:
reverted back to `alpha`